Quadriga/src/main/webapp/WEB-INF/views/workbench/addpublicpage.jsp

```diff
@@ -0,0 +1,151 @@
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
+<%@taglib uri="http://www.springframework.org/tags/form" prefix="form"%>
+
+<!-- Content -->
+<link rel="stylesheet"
+	href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
+<script>
+	function performAction(idValue) {
+		var data = {};
+		var titles = $("#title" + idValue).val();
```

Comment:
instead of explicitly getting elements by their id, try attaching the function to a button and get elements by going though the dom.

e.g.:
```
$('submit1').click(performAction())
```
inside performAction you should then be able to use ```$(this)``` to refer to the button and use ```find```, ```parent()```, etc. to navigate to the input elements you are looking for.